climateconnect/climateconnect
# Fix: Organization cards on landing page visual bug

## Description

Solves #1026 

- If an organization has more than 1 type, the chips will stack on top of each other rather than be next to each other
- The summary text now goes into the next line and will cut off like it does in /browse#organizations


## Test plan

- Visit the landing page
- Scroll down to "Find a climate action organization and get involved"
- The cards should be visible there

## Before landing

1. PR has meaningful title
1. `yarn lint` passes (frontend)
1. `yarn format` passes (frontend)
1. `make format` passes (backend)
